.github/workflows/size-limit.yml

```diff
@@ -0,0 +1,18 @@
+name: "size"
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  size:
+    runs-on: ubuntu-16.04
+    env:
+      CI_JOB_NUMBER: 1
+
+    steps:
```

Comment:
Is it possible to add a restore-from-cache step?

Reply:
Probably, but given the pains I've had getting size-limit to work, I want to add the restore from cache after I get size-limit working. Then I only have to debug one thing at a time.